Cargo.toml

```diff
@@ -101,13 +104,12 @@ clvm-utils = { path = "./crates/clvm-utils", version = "0.10.0" }
 clvm-derive = { path = "./crates/clvm-derive", version = "0.10.0" }
 chia-fuzz = { path = "./crates/chia-consensus/fuzz", version = "0.10.0" }
 blst = { version = "0.3.12", features = ["portable"] }
-clvmr = "0.7.0"
+clvmr = "0.8.0"
 syn = "2.0.70"
 quote = "1.0.32"
 proc-macro2 = "1.0.84"
 proc-macro-crate = "1.3.1"
 anyhow = "1.0.86"
-sha2 = "0.10.8"
```

Comment:
If we're still using this in a couple places for now, this will need to be here still.

Reply:
ok ill move back from workspace